src/components/RightSidebar/MoreMenu.astro

```diff
@@ -60,12 +61,20 @@ const showMoreSection = CONFIG.COMMUNITY_INVITE_URL
 					</svg>
 					<span>Contribute to the project</span>
 				</a>
-			</li>
+				</li>
+				<li class={`heading-link depth-2 git-btn`}>
```

Comment:
Instead of concatenating the strings like this, use the 3rd party solution like clsx.

Reply:
This is the same as provided by template from astro above

Reply:
If we want to use different approach it's out of the scope of this PR

Reply:
Okay, then let's keep it. 
However, I would like to introduce the clsx to the codebase to keep class names clean and concise.